<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsuarioCadastroPessoal</class>
 <widget class="QWidget" name="UsuarioCadastroPessoal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>791</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#voltar_button{
	background-color: rgba(0, 0, 0, 0);
	color: rgba(85, 98, 112, 255);
    border-radius: 20px;
    border: 1px solid rgba(255, 255, 255, 30);
    font-weight: bold;
}

QPushButton#voltar_button:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(25, 55, 100, 240),
        stop:1 rgba(220, 220, 220, 80)
    );
	color: rgba(155, 168, 182, 220);
}

QPushButton#voltar_button:pressed {
	padding-left: 5px;
	padding-top: 5px;
	color: rgba(115, 128, 142, 255);
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(10, 25, 50, 230),
        stop:1 rgba(180, 180, 180, 50)
    );
}



QPushButton#usu_cadastrar_button {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(15, 35, 70, 230),
        stop:1 rgba(200, 200, 200, 60)
    );
    color: rgba(255, 255, 255, 230);
    border-radius: 5px;
    border: 1px solid rgba(255, 255, 255, 30);
    font-weight: bold;
}

QPushButton#usu_cadastrar_button:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(25, 55, 100, 240),
        stop:1 rgba(220, 220, 220, 80)
    );
}

QPushButton#usu_cadastrar_button:pressed {
    padding-left: 3px;
    padding-top: 3px;
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(10, 25, 50, 230),
        stop:1 rgba(180, 180, 180, 50)
    );
}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>601</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/log_background.png);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fundotransparente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>581</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba( 0 ,0, 0, 100 );
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 214);</string>
    </property>
    <property name="text">
     <string>Cadastro de Usu√°rio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>69</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 214);</string>
    </property>
    <property name="text">
     <string>Dados pessoais</string>
    </property>
   </widget>
   <widget class="QPushButton" name="voltar_button">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>57</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Voltar</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/ic_voltar.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>150</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 214);</string>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usu_nome">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>178</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 80);
    border: 2px solid rgba(105, 118, 132, 220);
    border-radius: 10px;
    color: rgba(255, 255, 255, 230);
    padding: 6px 10px;
}

QLineEdit:focus {
    border: 2px solid rgba(130, 170, 255, 255);
    background-color: rgba(0, 0, 0, 100);
}
</string>
    </property>
    <property name="placeholderText">
     <string>Informe seu nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>241</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 214);</string>
    </property>
    <property name="text">
     <string>CPF:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usu_cpf">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>267</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 80);
    border: 2px solid rgba(105, 118, 132, 220);
    border-radius: 10px;
    color: rgba(255, 255, 255, 230);
    padding: 6px 10px;
}

QLineEdit:focus {
    border: 2px solid rgba(130, 170, 255, 255);
    background-color: rgba(0, 0, 0, 100);
}
</string>
    </property>
    <property name="placeholderText">
     <string>Insira seu CPF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usu_email">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>357</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 80);
    border: 2px solid rgba(105, 118, 132, 220);
    border-radius: 10px;
    color: rgba(255, 255, 255, 230);
    padding: 6px 10px;
}

QLineEdit:focus {
    border: 2px solid rgba(130, 170, 255, 255);
    background-color: rgba(0, 0, 0, 100);
}
</string>
    </property>
    <property name="placeholderText">
     <string>Informe email de contato</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>330</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 214);</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usu_senha">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>445</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 80);
    border: 2px solid rgba(105, 118, 132, 220);
    border-radius: 10px;
    color: rgba(255, 255, 255, 230);
    padding: 6px 10px;
}

QLineEdit:focus {
    border: 2px solid rgba(130, 170, 255, 255);
    background-color: rgba(0, 0, 0, 100);
}
</string>
    </property>
    <property name="placeholderText">
     <string>********</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>418</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 214);</string>
    </property>
    <property name="text">
     <string>Senha:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usu_senha_confirmar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>534</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 80);
    border: 2px solid rgba(105, 118, 132, 220);
    border-radius: 10px;
    color: rgba(255, 255, 255, 230);
    padding: 6px 10px;
}

QLineEdit:focus {
    border: 2px solid rgba(130, 170, 255, 255);
    background-color: rgba(0, 0, 0, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>********</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>507</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 214);</string>
    </property>
    <property name="text">
     <string>Confirmar senha:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="usu_cadastrar_button">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>635</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
